src/IntelliTect.Coalesce/TypeDefinition/PropertyViewModel.cs

```diff
@@ -608,7 +608,10 @@ public PropertySecurityInfo SecurityInfo
         {
             get
             {
-                var result = new PropertySecurityInfo();
+                var result = new PropertySecurityInfo(
+                    this.GetSecurityPermission<ReadAttribute>(),
+                    this.GetSecurityPermission<EditAttribute>(),
+                    this.GetSecurityPermission<DeleteAttribute>());
 
                 if (HasAttribute<ReadAttribute>())
```

Comment:
Lets move this chunk of code (the two `if (HasAttribute...` blocks) into the new constructor that takes the `SecurityPermission` objects.